ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py

```diff
@@ -2163,3 +2163,39 @@ def test_jax_numpy_real(
         test_values=True,
         val=x[0],
     )
+
+
+# inner
+@handle_frontend_test(
+    fn_tree="jax.numpy.inner",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_value=-10,
+        max_value=10,
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+)
+def test_jax_numpy_inner(
+    *,
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtypes, xs = dtype_and_x,
```

Comment:
Remove the comma after ``dtype_and_x``

Reply:
Ahh thanks for catching that, I've fixed this typo :)